hw3/hw3.py

```diff
@@ -1,101 +1,113 @@
-import os
 import sys
+from pyspark import StorageLevel
+from pyspark import keyword_only
+from pyspark.sql import SparkSession
+from pyspark.ml import Transformer
+
+
+# Create a spark session
+spark = SparkSession.builder \
+    .config("spark.jars",
+            "/Users/zack/Documents/SDSU/Fall 2022/BDA 602/src/bda602/hw3/mysql-connector-java-5.1.46/mysql-connector-java-5.1.46.jar") \
+    .master("local") \
+    .appName("HW3").getOrCreate()
+
+# Define Transformer Class
+
+class RollingAverageTransform(Transformer):
+
+    @keyword_only
+    def __init__(self):
+        super(RollingAverageTransform, self).__init__()
+        kwargs = self._input_kwargs
+        self.setParams(**kwargs)
+        return
+
+    @keyword_only
+    def setParams(self):
+        kwargs = self._input_kwargs
+        return self._set(**kwargs)
+
+    def _transform(data):
+        rolling_avg = rolling_average_calculation(spark, data)
+        return rolling_avg
+
+# Create a function to load the data from MariaDB
+def load_data(query):
```

Comment:
This is a great method to make for the object you made. Since you will have to use this method more than once for your calculations it makes the code look cleaner and easier to debug